stake-pool/single-pool/tests/helpers/token.rs

```diff
@@ -0,0 +1,54 @@
+#![allow(dead_code)]
+
+use {
+    mpl_token_metadata::{pda::find_metadata_account, state::Metadata},
+    solana_program_test::BanksClient,
+    solana_sdk::{
+        borsh::try_from_slice_unchecked,
+        hash::Hash,
+        message::Message,
+        program_pack::Pack,
+        pubkey::Pubkey,
+        signature::{Keypair, Signer},
+        transaction::Transaction,
+    },
+    spl_associated_token_account as atoken,
+    spl_token::state::{Account, Mint},
+};
+
+pub async fn create_ata(
+    banks_client: &mut BanksClient,
+    payer: &Keypair,
+    owner: &Pubkey,
+    recent_blockhash: &Hash,
+    pool_mint: &Pubkey,
+) {
+    #[allow(deprecated)]
+    let instruction = atoken::create_associated_token_account(&payer.pubkey(), owner, pool_mint);
```

Comment:
nit: If we don't use the new stuff, who will? :smiling_face_with_tear: 
```suggestion
    let instruction = atoken::instruction::create_associated_token_account(&payer.pubkey(), owner, pool_mint, &spl_token::id());
```